src/components/export-gist.jsx

```diff
@@ -0,0 +1,53 @@
+/* global XMLHttpRequest:false window:false */
+/* eslint-disable no-alert */
+import React from "react";
+
+export default class ExportGist extends React.Component {
+  parseJSON(responseText) {
+    try {
+      this.props.setMessage("");
```

Comment:
I'd move the "reset" of the message to the first line of `postGist`. It makes sense to reset the message in a central palce.